lib/block-supports/block-style-variations.php

```diff
@@ -273,7 +273,7 @@ function gutenberg_enqueue_block_style_variation_styles() {
 }
 
 // Add Gutenberg filters and action.
-add_filter( 'render_block_data', 'gutenberg_render_block_style_variation_support_styles', 10, 2 );
+add_filter( 'render_block_data', 'gutenberg_render_block_style_variation_support_styles', 10, 1 );
```

Comment:
Here and in other places `1` can be omitted as it's the default value. The same applies to the priority which also defaults to `10`.